c/tskit/tables.c

```diff
@@ -9190,6 +9338,37 @@ tsk_table_collection_subset(
             goto out;
         }
     }
+    if (keep_individuals) {
+        for (k = 0; k < (tsk_id_t) tables.individuals.num_rows; k++) {
+            if (individual_map[k] == TSK_NULL) {
+                ret = tsk_individual_table_get_row(&tables.individuals, k, &ind);
```

Comment:
Can use the ``unsafe`` versions of the the get_row functions here since we're in charge of the ID values.